Assignment6_docker/Homework2.sql

```diff
@@ -0,0 +1,48 @@
+-- *****************************************************************************
+--                                Assignment-2
+-- Host: localhost                                  Database: baseball
+-- Server version: 10.6.4-MariaDB-0ubuntu0.20.04
+--
+-- *****************************************************************************
+USE baseball;
```

Comment:
Probably this is not necessary since you've specified database in the rolling_avg.sh file.